cogs/Tips.py

```diff
@@ -48,6 +48,28 @@
 }
 
 
+async def send_tip(
+    ctx: ApplicationContext, tip: str, ping: Member | None = None, anonymous: str = "No"
+) -> None:
+    message_content = None if ping is None else ping.mention
+    embed = EmbedBuilder(
+        title=f"Tip: {tip.capitalize()}.",
+        description=TIPS[tip],
+        color=0x32DC64,  # a nice pastel green
+    ).build()
+
+    if anonymous.casefold() == "yes":
+        await ctx.send(message_content, embed=embed)
+        await ctx.respond(
+            "Thanks for the tip! It was sent anonymously.",
+            ephemeral=True,
+            delete_after=5,
+        )
+    else:
+        await ctx.respond(message_content, embed=embed)
+    Log(f"$ used tip: {tip} | in channel {ctx.channel.name}", ctx.author)
```

Comment:
I would suggest moving this log back into the command itself so that `DetectCode` doesn't trigger `used tip` logs, instead a new log line should be added to `DetectCode`